Resources/views/themes/admin/dashboard/block/ez.html.twig

```diff
@@ -0,0 +1,140 @@
+{% trans_default_domain 'dashboard' %}
+
+{#
+  Want to edit these messages?
+  If you are on GPL, as always make sure to share your modifications, as well as your bundles. But to
+  make sure everyone benefits, please consider contributing modifications to ezsystems/ez-support-tools.
+#}
+
+{% set badge = "" %}
+{% set severity = 0 %}
+{% set levels = {0: "info", 1: "warning", 2: "danger"} %}
+{% set icons = {0: "", 1: "&#9888;", 2: "&#9888;"} %}
+{% set status %}
+{% set urls = {
+    'contact': "https://ez.no/About-eZ/Contact-Us",
+    'license': "https://ez.no/About-our-Software/Licenses-and-agreements",
+    'ttl': "https://ez.no/About-our-Software/Licenses-and-agreements/eZ-Trial-and-Test-License-Agreement-eZ-TTL-v2.1",
+    'service_life': "https://support.ez.no/Public/Service-Life",
+    'support_service': "https://ez.no/Services/Support-Maintenance",
+    'training_service': "https://ez.no/Services/Training",
+    'consulting_service': "https://ez.no/Services/Consulting",
+    'ee_product': "https://ez.no/Products/eZ-Platform-Enterprise-Edition",
+} %}
+{% spaceless %}
+    {% if not ez.release %}
+        {% set severity = 1 %}
+        <div class="alert alert-warning collapse" role="alert" id="systemInfoCollapse">
+            {% trans %}
+                The system could not find your <code>composer.lock</code> file, it's needed in order to determine information about
+                your eZ install, and recommended to be kept on project development to make sure same package versions are used across all environments.
```

Comment:
```suggestion
                your eZ Platform installation. We recommend to keep this file during project development to make sure same package versions are used across all environments.
```

Reply:
hmm, this now kind of implies we only need this file during development, but actually we need it also after code is deployed in prod. Could we write this in a different way maybe?